script/cibuild

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
maybe we should start replacing this by #!/usr/bin/env bash ?

Reply:
👍

Reply:
well, its considered now a best practice: if runs bash no matter where is localted